wpilibc/src/main/native/cpp/geometry/Pose2d.cpp

```diff
@@ -40,7 +40,7 @@ Pose2d Pose2d::RelativeTo(const Pose2d& other) const {
 Pose2d Pose2d::Exp(const Twist2d& twist) const {
   const auto dx = twist.dx;
   const auto dy = twist.dy;
-  const auto dtheta = twist.dtheta;
+  const auto dtheta = twist.dtheta.to<double>();
 
   const auto sinTheta = std::sin(dtheta);
```

Comment:
I'm curious about whether doing `const auto dtheta = twist.dtheta;` and `const auto sinTheta = units::math::sin(dtheta);` works.

Reply:
This isn't worth doing because `dtheta` will have to be converted to a double anyway for the later calculations.